pyproject.toml

```diff
@@ -0,0 +1,128 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "syne-tune"
+dynamic = ["version"]
+description = "Distributed Hyperparameter Optimization"
+readme = "README.md"
+authors = [{name = "AWS"}]
+license = {text = "Apache-2.0"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "numpy>=1.22.0, <1.27.0",
+    "dill>=0.3.6",
+    "pandas",
+    "typing_extensions",
+    "sortedcontainers",
+]
+
+[project.optional-dependencies]
+gpsearchers = [
+    "scipy<1.15.3",
+    "autograd>=1.3",
+]
+kde = [
+    "statsmodels",
+]
+dev = [
+    "pytest",
+    "pytest-cov~=4.1.0",
+    "pytest-timeout",
+    "black==22.3.0",
+    "flake8",
+    "sphinx<8.0.0",
+    "nbsphinx",
+    "ipykernel",
+    "pypandoc_binary",
+    "sphinx-rtd-theme",
+    "sphinx-autodoc-typehints",
+    "traitlets<=5.11.2",
+    "myst-parser",
+    "sphinx_copybutton",
+    "sphinxcontrib-bibtex",
+    "sphinxcontrib.jquery",
+    "wheel",
+]
+aws = [
+    "boto3",
+    "sagemaker>=2.112.0",
+    "PyYaml",
+    "ujson",
+    "s3fs",
+]
```

Comment:
should we remove this? (we should grep the doc then to avoid dangling references)
```suggestion
```